docs/git-branches-conventions.md

```diff
@@ -0,0 +1,59 @@
+# Git branch conventions
+
+There are 4 types of branches:
+
+1. The master branch
```

Comment:
```suggestion
1. The master/main branch
```
Quite a few projects have moved to `main` and I recall our repositories are sometimes using `master` and sometimes `main`